docker-compose.volumes.yml

```
@@ -32,5 +32,8 @@ services:
 
 volumes:
   voice-memos:
+    name: wafflebot-${ENVIRONMENT}-voice-memos
```

devinai I'm getting this error:
```
./run-wafflebot.sh staging

+ docker compose -f docker-compose.yml -f docker-compose.volumes.yml -f docker-compose.staging.yml run --rm file-downloader
WARN[0000] The "ENVIRONMENT" variable is not set. Defaulting to a blank string.
WARN[0000] The "ENVIRONMENT" variable is not set. Defaulting to a blank string.
WARN[0000] The "ENVIRONMENT" variable is not set. Defaulting to a blank string.
```

---

devinai can we fail here if ENVIRONMENT is not set?